examples/typescript/aws-s3-static-website/website/index.html

```diff
@@ -0,0 +1,9 @@
+<html xmlns="http://www.w3.org/1999/xhtml" >
```

Comment:
Let's use modern HTML5 for this page as well.
```suggestion
<!DOCTYPE html>
<html>
```